cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/runtimejob/DataprocRuntimeJobManager.java

```diff
@@ -189,9 +189,21 @@ public void launch(RuntimeJobInfo runtimeJobInfo) throws Exception {
       // step 4: submit hadoop job to dataproc
       Job job = getJobControllerClient().submitJob(request);
       LOG.debug("Successfully submitted hadoop job {} to cluster {}.", job.getReference().getJobId(), clusterName);
+      DataprocUtils.emitMetric(provisionerContext, region, StatusCode.Code.OK,
+                               // Constants.Metrics.Provisioner.SUBMIT_JOB_COUNT =
+                               "provisioner.submitJob.response.count");
     } catch (Exception e) {
       // delete all uploaded gcs files in case of exception
       DataprocUtils.deleteGCSPath(getStorageClient(), bucket, runRootPath);
+      StatusCode.Code statusCode = StatusCode.Code.INTERNAL;
```

Comment:
This segment of code has repeated many times. Better to refactor it out to the DataprocUtils method.

Reply:
fixed.